prow/config/presubmits.yaml

```diff
@@ -116,6 +116,29 @@ presubmits:
         securityContext:
           privileged: true
   Maistra/istio:
+  - name: unittests
```

Comment:
Thia should probably be renamed to `unittests-1.2` and the current one below to `1.1`

Reply:
that's correct, I missed that

Reply:
it looks like they don't have to be unique though: https://github.com/Maistra/istio/pull/113

Reply:
Yeah, I wouldn't expect them to be unique. I think a common way to organize these is actually one (very similar) file per branch, where each file has mostly identical configurations.

Reply:
oh well. apparently the `init` target doesn't work without an interactive shell anyway. we might not need it anymore

Reply:
Although they don't *need* to be unique, it's good to have them to be different, for organization and to avoid future clashes (e.g., if I type `/test unittests`, which one will run?)

Reply:
I'm happy to change it, but that will be different from I think every other Prow setup I've seen. What about your question in reverse? Why should I have to type something different on certain PRs to run the tests?

Reply:
Ok, so, if the jobs are the same, why not just add the branches into the current, existing `unittests` job? Why duplicate it?

Reply:
I believe badges are tied to using the name as well. Not sure what else this breaks.

Reply:
> Ok, so, if the jobs are the same, why not just add the branches into the current, existing unittests job? Why duplicate it?

It's not 100% the same. The image is updated to the 1.2 tag. Anyway, I think I'm misremembering. I guess there's no need for the names to be the same as long as the `context` and `rerun_command` bits make sense. I will just put up a PR to change the name and set those. We should probably look at splitting these up by branch as we have more branches and jobs.